completion/available/gem.completion.bash

```diff
@@ -1,41 +1,34 @@
-#!/usr/bin/env bash
+# shellcheck shell=bash
 # Completion for gem
 
-_installcomp() {
-  if [ -z "$REMOTE_GEMS" ]
-  then
-    REMOTE_GEMS=( $(gem list --remote --no-versions | tr '\n' ' ') )
-  fi
+__gem_completions() {
```

Comment:
could we drop the plural to `__gem_completion`?

Reply:
fixed, nice idea!